### ISSUE TITLE
Can you help me here again? I have updated the example https://codesandbox.io/s/lucid-fire-jq1nj?file=/src/basic.js

### ISSUE THREAD

**Issue by anubra266** [anubra266/choc-autocomplete#89]
Can you help me here again? I have updated the example https://codesandbox.io/s/lucid-fire-jq1nj?file=/src/basic.js

I'm trying to make work the component with a default value using `label` and `value`.
I have added the `value` prop to the input component to be able to update the value of the input with the `reset` function of RHF

The problem is when you change to other option it will let the input in blank and work normally since the second selection

Steps to reproduce the problem

1. Select a diferent option than the default value (it will be blank)
2. Select a diferent option for the second time (it will work)

_Originally posted by @gepd in https://github.com/anubra266/choc-autocomplete/issues/77#issuecomment-946320781_

**Comment by gepd** [comment]
Hey @anubra266 I think you are very closer to solve this problem, I can see now the value in the input, however If I pass an object the value of `AutocompleteInput` component will be `[object Object]`

If I do

```js
<AutoComplete
          value={value.label}
          onChange={a => {
            setValue(a);
          }}
```
I'll have a type error (I'm using typescript)

I shouldn't modify onChange as problem is in the first render. Maybe the `AutocompleteInput` should have a new prop `selectDisplay` with a callback to solve this problem

```js
<AutoComplete
          value={value}
          onChange={a => {
            setValue(a);
          }}>
  <AutocompleteInput selectDisplay={value => value.label} />
</Autocomplete>
```

**Comment by anubra266** [comment]
@gepd, you don't need to pass an object ast the value, if value of your `AutoCompleteItem`s are objects, just pass the value key as the value of the `AutoComplete`, it'll automatically be resolved to match the Item that has that value as it's `value` key

**Comment by gepd** [comment]
I have updated the first example and is not working, as you can see in the first render, I keep seing the `value` and even when you press the reset button it shows the given value, you can verify in the console after press submit

**Comment by anubra266** [comment]
@gepd Fixed in [v4.11.1](https://github.com/anubra266/choc-autocomplete/releases/tag/v4.11.1)
